ape_vyper/compiler.py

```diff
@@ -112,7 +112,10 @@ def compile(self, contract_filepaths: List[Path]) -> List[ContractType]:
                     vyper_version=vyper_version,
                 )["<stdin>"]
             except Exception as e:
-                raise Abort(f"Unable to compile {path} with Vyper version: {vyper_version}\n\n{e.stderr_data}") from e
+                raise Abort(
+                    f"Unable to compile {path} with Vyper version: \
+                        {vyper_version}\n\n{e.stderr_data}"  # type: ignore
```

Comment:
This works to concatenate strings
```suggestion
                    f"Unable to compile {path} with Vyper version: {vyper_version}"
                    f"\n\n{e.stderr_data}"
```